src/main/java/org/ice4j/socket/jdk8/DelegatingServerSocketChannel.java

```diff
@@ -54,6 +75,37 @@ public DelegatingServerSocketChannel(T delegate)
 
         delegateAsSelChImpl
             = (delegate instanceof SelChImpl) ? (SelChImpl) delegate : null;
+
+        Method method;
+        try
+        {
+            method = SelChImpl.class.getMethod(
```

Comment:
Do we want to the the method from `SelChImpl`, or from the class of our delegate? I.e. if the delegate overrides the method, which one should we call?

Reply:
I need to test it, as there was some problem which I do not recall well when getting the class from the instance ...

Reply:
It doesn't work obtaining the method from the instance with delegateAsSelChImpl.getClass().getMethod. The error:
java.lang.IllegalAccessException: class org.ice4j.socket.jdk8.DelegatingServerSocketChannel cannot access a member of class sun.nio.ch.ServerSocketChannelImpl (in module java.base) with modifiers "public"

With obtaining the method from the class it runs with one warning in the beginning:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.ice4j.socket.jdk8.DelegatingServerSocketChannel (file:/usr/share/jitsi-videobridge/lib/ice4j-2.0.0-SNAPSHOT.jar) to method sun.nio.ch.SelChImpl.translateInterestOps(int)
WARNING: Please consider reporting this to the maintainers of org.ice4j.socket.jdk8.DelegatingServerSocketChannel
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

```

Reply:
I get this in the startup log (Illegal reflective access operation) with openjdk 11.0.6 on Ubuntu18.04